SavingsManagementSystem.Service/User/Implementations/AdminService.cs

```diff
@@ -35,5 +44,41 @@ public async Task<RegistrationResponse> RegisterAsync(RegistrationRequest reques
 
 			return response;
 		}
+
+
+		public async Task SendMemberInviteAsync(string email)
+		{
+			//create a verification token for the link
+			var vToken = await _vTokenService.CreateVerificationTokenAsync(30, "Successful", email);
+			if (vToken == null)
```

Comment:
how possible can vtoken is null when you aren't verify anything, you are just adding to database.

Reply:
remove the null check